dev-lang/php/patches/php-7.1.8.patchset

```diff
@@ -0,0 +1,481 @@
+-      if test -r $i/include/bzlib.h; then
++    for i in /usr/local /usr /system; do
++      if test -r $i/develop/headers/bzlib.h; then
+         BZIP_DIR=$i
+         AC_MSG_RESULT(found in $i)
+         break
+@@ -26,7 +26,7 @@ if test "$PHP_BZ2" != "no"; then
+ 
+   PHP_CHECK_LIBRARY(bz2, BZ2_bzerror, 
+   [
+-    PHP_ADD_INCLUDE($BZIP_DIR/include)
++    PHP_ADD_INCLUDE($BZIP_DIR/develop/headers)
+     PHP_ADD_LIBRARY_WITH_PATH(bz2, $BZIP_DIR/$PHP_LIBDIR, BZ2_SHARED_LIBADD)
+     AC_DEFINE(HAVE_BZ2,1,[ ])
+   ], [
+diff --git a/ext/curl/config.m4 b/ext/curl/config.m4
+index 2f82c34..b6a09d9 100644
+--- a/ext/curl/config.m4
++++ b/ext/curl/config.m4
+@@ -10,8 +10,8 @@ if test "$PHP_CURL" != "no"; then
+     CURL_DIR=$PHP_CURL
+   else
+     AC_MSG_CHECKING(for cURL in default path)
+-    for i in /usr/local /usr; do
+-      if test -r $i/include/curl/easy.h; then
++    for i in /usr/local /usr /system; do
++      if test -r $i/develop/headers/curl/easy.h; then
+         CURL_DIR=$i
+         AC_MSG_RESULT(found in $i)
+         break
+@@ -22,7 +22,7 @@ if test "$PHP_CURL" != "no"; then
+   if test -z "$CURL_DIR"; then
+     AC_MSG_RESULT(not found)
+     AC_MSG_ERROR(Please reinstall the libcurl distribution -
+-    easy.h should be in <curl-dir>/include/curl/)
++    easy.h should be in <curl-dir>/develop/headers/curl/)
+   fi
+ 
+   CURL_CONFIG="curl-config"
+diff --git a/ext/gd/config.m4 b/ext/gd/config.m4
+index 1bc5f98..46f9db2 100644
+--- a/ext/gd/config.m4
++++ b/ext/gd/config.m4
+@@ -47,12 +47,12 @@ dnl
+ 
+ AC_DEFUN([PHP_GD_ZLIB],[
+ 	if test "$PHP_ZLIB_DIR" != "no" && test "$PHP_ZLIB_DIR" != "yes"; then
+-		if test -f "$PHP_ZLIB_DIR/include/zlib/zlib.h"; then
++		if test -f "$PHP_ZLIB_DIR/develop/headers/zlib/zlib.h"; then
+ 			PHP_ZLIB_DIR="$PHP_ZLIB_DIR"
+-			PHP_ZLIB_INCDIR="$PHP_ZLIB_DIR/include/zlib"
+-		elif test -f "$PHP_ZLIB_DIR/include/zlib.h"; then
++			PHP_ZLIB_INCDIR="$PHP_ZLIB_DIR/develop/headers/zlib"
++		elif test -f "$PHP_ZLIB_DIR/develop/headers/zlib.h"; then
+ 			PHP_ZLIB_DIR="$PHP_ZLIB_DIR"
+-			PHP_ZLIB_INCDIR="$PHP_ZLIB_DIR/include"
++			PHP_ZLIB_INCDIR="$PHP_ZLIB_DIR/develop/headers"
+ 		else
+ 			AC_MSG_ERROR([Can't find zlib headers under "$PHP_ZLIB_DIR"])
+ 		fi
+@@ -72,16 +72,16 @@ AC_DEFUN([PHP_GD_ZLIB],[
+ AC_DEFUN([PHP_GD_WEBP],[
+   if test "$PHP_WEBP_DIR" != "no"; then
+ 
+-    for i in $PHP_WEBP_DIR /usr/local /usr; do
+-      test -f $i/include/webp/decode.h && GD_WEBP_DIR=$i && break
++    for i in $PHP_WEBP_DIR /usr/local /usr /system; do
++      test -f $i/develop/headers/webp/decode.h && GD_WEBP_DIR=$i && break
+     done
+ 
+     if test -z "$GD_WEBP_DIR"; then
+       AC_MSG_ERROR([webp/decode.h not found.])
+     fi
+ 
+-    for i in $PHP_WEBP_DIR /usr/local /usr; do
+-      test -f $i/include/webp/encode.h && GD_WEBP_DIR=$i && break
++    for i in $PHP_WEBP_DIR /usr/local /usr /system; do
++      test -f $i/develop/headers/webp/encode.h && GD_WEBP_DIR=$i && break
+     done
```

Comment:
You could also leave the loop as is, and add another check after the loop, this would be more acceptable upstream.
````
for i in $PHP_WEBP_DIR /usr/local /usr; do
   test -f $i/include/webp/encode.h && GD_WEBP_DIR=$i && break
done
test -f /system/develop/headers/webp/encode.h && GD_WEBP_DIR=/system
````